utils/db_utils.py

```diff
@@ -104,23 +104,37 @@ def query_confirmed_trips(start_date, end_date):
 def query_demographics():
     logging.debug("Querying the demographics for (no date range)")
     ts = esta.TimeSeries.get_aggregate_time_series()
```

Comment:
you should add a comment here highlighting the different return type here since it is not the same as the other dataframes.